model/riscv_sstc.sail

```diff
@@ -0,0 +1,20 @@
+/*=======================================================================================*/
+/*  This Sail RISC-V architecture model, comprising all files and                        */
+/*  directories except where otherwise noted is subject the BSD                          */
+/*  two-clause license in the LICENSE file.                                              */
+/*                                                                                       */
+/*  SPDX-License-Identifier: BSD-2-Clause                                                */
+/*=======================================================================================*/
+
+/* Sstc - supervisor time compare */
+mapping clause csr_name_map = 0x14D  <-> "stimecmp"
+mapping clause csr_name_map = 0x15D  <-> "stimecmph"
+
+function clause is_CSR_defined(0x14D) = extensionEnabled(Ext_S) & extensionEnabled(Ext_Sstc)
+function clause is_CSR_defined(0x15D) = extensionEnabled(Ext_S) & extensionEnabled(Ext_Sstc) & xlen == 32
+
+function clause read_CSR(0x14D) = stimecmp[sizeof(xlen) - 1 .. 0]
```

Comment:
Now we have [this commit](https://github.com/riscv/sail-riscv/commit/ba5f6d0ee4cb7168499e5ddb739f2685ad3469fd), I think this can just be `xlen - 1`?

Reply:
Good spot - updated.